hack/deploy/controlplane/deployment.yaml

```diff
@@ -28,7 +28,6 @@ spec:
           - "--enable-bootstrap-token-auth"
           - "--enable-priority-and-fairness=false"
           - "--api-audiences="    
-          - "--storage-backend=etcd3"
```

Comment:
Need send a request to cicd for creating quay.io/stolostron/multicluster-controlplane